charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml

```diff
@@ -148,6 +148,10 @@ spec:
           - name: SSL_CERT_DIR
             value: /tmp/certs
           {{- end }}
+          {{- if eq .Release.IsInstall true }}
+          - name: QUERY_MONGO_DB
```

Comment:
Is this only for  middleware use case? If yes, should we rename this to be more specific to that use case?

Reply:
Yes, it only for middleware . This ENV is part of the middleware codebase . So i am reluctant in changing it